contrib/python-desktop-notifier/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "python-desktop-notifier"
+pkgver = "5.0.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+depends = ["python", "python-dbus-next"]
+checkdepends = ["python-pytest", "python-pytest-asyncio", *depends]
+pkgdesc = "Cross-platform desktop notification library"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "MIT"
+url = "https://pypi.org/project/desktop-notifier"
+source = f"https://github.com/samschott/desktop-notifier/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "2d756fb4e82b8f95edb0e49ec9a3e8e4d5ba404c194cc6bffac14cd8bad33f74"
+# no dbus session bus in chroot
+options = ["!check"]
```

Comment:
also try enable check here

Reply:
I suspect this isn't going to work because the chroot doesn't have a DE to provide `org.freedesktop.Notifications`